homeassistant/components/husqvarna_automower/strings.json

```diff
@@ -5,6 +5,10 @@
         "title": "[%key:common::config_flow::title::reauth%]",
         "description": "The Husqvarna Automower integration needs to re-authenticate your account"
       },
+      "missing_scope": {
+        "title": "Your account is missing some API conenctions.",
+        "description": "For the best experience with this integration both `Authentiacation API` and `Automower Connect API` should be connected. Please make sure, that both of them are conncted to your account in the [Husqvarana Developers Portal](https://developer.husqvarnagroup.cloud/applications/{client_id}). You can find more inforamtion [here](https://www.home-assistant.io/integrations/husqvarna_automower/)."
```

Comment:
```suggestion
        "description": "For the best experience with this integration both the `Authentication API` and the `Automower Connect API` should be connected. Please make sure that both of them are connected to your account in the [Husqvarna Developer Portal](https://developer.husqvarnagroup.cloud/applications/{client_id}). You can find more information [here](https://www.home-assistant.io/integrations/husqvarna_automower/)."
```

Reply:
The urls should be passed as placeholder descriptions and not directly in the translation.

Reply:
Done in https://github.com/home-assistant/core/pull/117098/commits/f43933fa1a9df65eeff6d18fbfd33648334d7b9f.
I've also simplified the messages. In the abort message we don't need the link again.